Dockerfile.no-systemd

```diff
@@ -0,0 +1,36 @@
+FROM balenalib/amd64-debian:buster-run
+
+ENV TERM xterm
+
+# Install ops/sre related packages.
+RUN install_packages \
+		htop \
+		jq \
+		nano \
+		netcat \
+		net-tools \
+		ifupdown \
+		strace \
+		vim \
+		wget
+# And configure them.
+COPY src/htoprc /root/.config/htop/
+
+# We use NodeJs everywhere.
+ENV NODE_VERSION 10.16.3
+ENV NPM_VERSION 6.10.3
+RUN curl -SL "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz" | tar xz -C /usr/local --strip-components=1 \
+	&& npm install -g npm@"$NPM_VERSION" \
+	&& npm cache clear --force \
+	&& rm -rf /tmp/*
+
+# Confd binary will be removed once we migrate to a different configuration management.
+ENV CONFD_VERSION 0.16.0
+RUN wget -O /usr/local/bin/confd https://github.com/kelseyhightower/confd/releases/download/v${CONFD_VERSION}/confd-${CONFD_VERSION}-linux-amd64 \
+	&& chmod a+x /usr/local/bin/confd
+ONBUILD COPY config/confd /etc/confd
+RUN mkdir /balena
```

Comment:
I'm testing with node base images for now. But will change if we decide to not go with that variant.

Reply:
+1, probably move this to the `RUN` instruction in line 5 so we don't have one extra layer.

Reply:
I don't get what is wrong with the current variant...
Coupling this with packages installation does not make much sense to me.

Reply:
@roman-mazur `RUN mkdir /balena` could be moved over line 5/ The comments view on gh pr seems confusing on this one.

Reply:
this way we would bundle it with packages installation, which is basically absolutely separate piece...
Is there indeed a big win on the caching side to break some logical organization?

Reply:
Have to say, I'm in agreement with @roman-mazur here. I'd rather see a logical order than shoving it in the package installation, purely because that implies it's part of the installation process. We do things in single `RUN`s quite a lot, but they're always related to that instruction, such as tearing down caches for installation or cleanup of modules that aren't required. I don't think the small overhead save is worth the change.